tool.go

```diff
@@ -55,7 +55,7 @@ func main() {
 	flagWatch := pflag.Lookup("watch")
 	pflag.BoolVarP(&options.Minify, "minify", "m", false, "minify generated code")
 	flagMinify := pflag.Lookup("minify")
-	pflag.BoolVar(&options.Color, "color", terminal.IsTerminal(2) && os.Getenv("TERM") != "dumb", "colored output")
+	pflag.BoolVar(&options.Color, "color", terminal.IsTerminal(syscall.Stderr) && os.Getenv("TERM") != "dumb", "colored output")
```

Comment:
I see, sorry about that.

I was a little surprised that was the case because `os.Stderr` is defined exactly the same on _all_ platforms:

http://gotools.org/os#Stderr

But I see that the type of `syscall.Stderr` is not `int` on Windows, because it's defined as:

```
Stderr = getStdHandle(STD_ERROR_HANDLE)
```

And `getStdHandle` returns type `Handle` which is defined as `uintptr`. So I should've done `terminal.IsTerminal(int(syscall.Stderr))` to make it work everywhere. But I guess `os.Stderr.Fd()` is also valid (and maybe simpler/easier to understand, since more people are familiar with `os` package than `syscall` package).

Thanks!